requirements-dev.txt

```diff
@@ -10,29 +10,20 @@
 # Installation toolchain
 setuptools~=68.2
 
-# Copyright linting toolchain
-reuse~=2.1
-
 # Documentation toolchain
 pydeps~=1.12.1
 myst-parser~=2.0.0
 sphinx~=7.2.6
 m2r2~=0.3.3.post2
 
 # Python Linting toolchain
-black>=23.1.0
-flake8>=6.0.0
-pylint>=2.16.0
 pytest-github-actions-annotate-failures>=0.1.7
```

Comment:
Is this still used? Or does out annotations stuff take care of this exclusively?

Reply:
It should be taken care of either there or in tools.

Some more deps should not be required. I'll pare the file back.

Reply:
Pare back complete